python/examples/tutorials/Data_Validation_Tutorial.ipynb

```diff
@@ -0,0 +1,4235 @@
+              "        line-height: 140%;\n",
+              "        color: #FFFFFF;\n",
+              "      }\n",
+              "\n",
+              "      .turquoise-background-color {\n",
+              "        background-color: #1DBB42;\n",
+              "      }\n",
+              "\n",
+              "      .bordeaux-background-color {\n",
+              "        background-color: #C6462A;\n",
+              "      }\n",
+              "\n",
+              "      .border-solid-gray {\n",
+              "        border: 1px solid #CED4DA;\n",
+              "        border-radius: 4px;\n",
+              "      }\n",
+              "\n",
+              "      .display-flex {\n",
+              "        display: flex;\n",
+              "      }\n",
+              "\n",
+              "      .justify-content-space-between {\n",
+              "        justify-content: space-between;\n",
+              "      }\n",
+              "\n",
+              "      .justify-content-center {\n",
+              "        justify-content: center;\n",
+              "      }\n",
+              "\n",
+              "      .align-items-center {\n",
+              "        align-items: center;\n",
+              "      }\n",
+              "\n",
+              "      .align-items-flex-start {\n",
+              "        align-items: flex-start;\n",
+              "      }\n",
+              "\n",
+              "      .padding-right-30 {\n",
+              "        padding-right: 30px;\n",
+              "      }\n",
+              "\n",
+              "      .notif-circle-container{\n",
+              "        position: absolute;\n",
+              "        top: 25px;\n",
+              "        right: 25px;\n",
+              "        padding: 5.3px;\n",
+              "        border-radius: 50%;\n",
+              "        background-color: white;\n",
+              "        cursor: pointer;\n",
+              "      }\n",
+              "\n",
+              "      .notif-circle {\n",
+              "        position: absolute;\n",
+              "        top: 2px;\n",
+              "        right: 2px;\n",
+              "        padding: 3.3px;\n",
+              "        border-radius: 50%;\n",
+              "        background-color: #F2994A;\n",
+              "      }\n",
+              "\n",
+              "      .alert-list-text {\n",
+              "        width: 70%\n",
+              "      }\n",
+              "\n",
+              "     @media screen and (min-width: 500px) {\n",
+              "       .desktop-content {\n",
+              "         display: block;\n",
+              "       }\n",
+              "       .no-responsive {\n",
+              "         display: none;\n",
+              "       }\n",
+              "     }\n",
+              "    &lt;/style&gt;\n",
+              "  &lt;/head&gt;\n",
+              "\n",
+              "  &lt;body id=&quot;generated-html&quot;&gt;&lt;/body&gt;\n",
+      
```

Comment:
We could mention here that we're aiming to `s/validator/condition/` and conditions can run actions too.